### DISCUSSION TITLE
Automatic translation of multiline string... translates multilines, not the whole string

### DISCUSSION THREAD

**Discussion by sebpeterson** [WeblateOrg/weblate#11944] [General] [answered]
Hi ,
I am new to Weblate and enjoying it so far ! Congrats!

Ran into this issue, trying to translate multiline strings, such as:

msgid ""
"The ISSB standard, or International Sustainability Standards Board standard,\n"
"is a set of globally-recognized rules for companies to disclose sustainability-related\n"
"information to investors. These standards aim to provide consistent, comparable, and\n"
"decision-useful data across companies and countries. "
msgstr ""
"La norme ISSB, ou norme International Sustainability Standards Board,\n"
" est un ensemble de règles mondialement reconnues permettant aux entreprises "
"de divulguer leurs informations en matière de développement durable.\n"
" informations aux investisseurs. Ces normes visent à fournir des "
"informations cohérentes, comparables et\n"
" données utiles à la décision dans les entreprises et les pays."

When I do an automated translation, I wil get in return the translation of individual lines... not the actual string, which is killing me. I can only imagine this is a know issue?

Can anybody help me out here?
Cheers.

**Comment by nijel** [comment]
Is the string displayed correctly in Weblate?

What automatic suggestion service do you use?

**Reply by sebpeterson** [reply]
Using "Google Cloud Translation Advanced" 

Here is what is displayed:
![image](https://github.com/WeblateOrg/weblate/assets/66732129/e39fff79-1f58-46ac-b6b6-60aa092f1e3e)

**Reply by sebpeterson** [reply]
I guess weblate is sending source as separate lines, and that is how it is treated by Google...

**Reply by nijel** [reply]
Yes, newlines are preserved, see https://github.com/WeblateOrg/weblate/issues/10145

**Reply by sebpeterson** [reply, marked as answer]
Just discovered the "trimmed" option for blocktranslate in Django... which solved my issue.

{% blocktranslate trimmed %}

So not a weblate issue actually !
Cheers
